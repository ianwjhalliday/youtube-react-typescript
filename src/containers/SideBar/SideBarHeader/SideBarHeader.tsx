import React from 'react';
import {Menu} from 'semantic-ui-react';
import './SideBarHeader.scss';

export interface SideBarHeaderProps {
  title: string;
}

export function SideBarHeader(props: SideBarHeaderProps) {
  const heading = props.title.toUpperCase();
  return (
    <Menu.Item>
      <Menu.Header className="sidebar-header">{heading}</Menu.Header>
    </Menu.Item>
  );
}
