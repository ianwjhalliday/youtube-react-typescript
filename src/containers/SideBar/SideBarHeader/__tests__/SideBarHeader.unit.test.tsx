import React from 'react';
import {shallow} from 'enzyme';
import SideBarHeader from '../SideBarHeader';

test('renders SideBarHeader with uppercase title', () => {
  const wrapper = shallow(
    <SideBarHeader title="Header" />
  );
  expect(wrapper).toMatchSnapshot();
});
