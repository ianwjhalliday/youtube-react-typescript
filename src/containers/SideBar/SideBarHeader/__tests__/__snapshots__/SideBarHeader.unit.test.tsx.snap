// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SideBarHeader with uppercase title 1`] = `
<MenuItem>
  <MenuHeader
    className="sidebar-header"
  >
    HEADER
  </MenuHeader>
</MenuItem>
`;
